src/hooks/use-database.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import { useContext } from 'react'
+import { DatabaseContext } from '../DatabaseProvider'
+import Database from '../Database'
+import invariant from '../utils/common/invariant'
+
+import type DatabaseType from '../Database'
+
+export function useDatabase(): DatabaseType {
```

Comment:
```suggestion
export function useDatabase(): Database {
```

You're already importing Database - and its type is the same